assets/dev/scss/editor/_buttons-mixin.scss

```diff
@@ -0,0 +1,117 @@
+@mixin e-button {
```

Comment:
I think this file should be in the global folder since you are using it globally.

Reply:
I agree. Moved.